test/Data/Time/PreciseDuration.purs

```diff
@@ -22,27 +23,27 @@ test
   -> Decimal
   -> Array Decimal
   -> Aff r Unit
-test fn ctr div = traverse_ \input -> do
+test fn ctr div = traverse_ \(input :: Decimal) -> do
```

Comment:
Is the inline annotation here necessary?

Reply:
It was to help me figure out a type error...